src/Dnn.vNext/Pages/_Admin.cshtml

```diff
@@ -95,16 +95,73 @@
                     url: 'api/page/EditPage',
                     type: 'POST',
                     success: function (data) {
+                        $('#persona-edit-button').hide();
+                        $('#persona-cancel-edit-button').show();
+
                         if ($('footer').length) {
                             $('footer').remove();
                         }
                         else {
-                            $('#main').append(data);    
-                        }                        
+                            $('#main').append(data);
+                        }
                     }
                 });
             });
         });
+
+        $(function () {
+            $('#persona-cancel-edit-button').click(function () {
+                $('#persona-edit-button').show();
+                $('#persona-cancel-edit-button').hide();
+                $('footer').remove();
+            });
+        });
+    </script>
+
+
+    <script type="text/javascript">
+        var closeAllMenus = function(){
```

Comment:
This is now a global variable, was that intentional? I would wrap this inside of `$(function() { });` or `$(document).ready(function() { });` so the variable isn't a global. the 2nd option here using the ready function is the preferred approach I need to go through and update the other instances throughout the code.

Reply:
Not intentional... have updated it for my script, but you may want to move it to a higher level as you noted.